src/script/interpreter.cpp

```diff
@@ -263,6 +262,10 @@ bool EvalScript(std::vector<valtype> &stack, const CScript &script,
                 return set_error(serror, ScriptError::DISABLED_OPCODE);
             }
 
+            if (opcode == OP_SCRIPTTYPE) {
```

Comment:
You're setting it up to delete this in a future commit where taproot is enabled?

Reply:
Nevermind. This is going to be a pattern.